cmd/serve.go

```diff
@@ -108,24 +109,14 @@ func StartServer(logger *log.Zap, cfg *config.Shield) error {
 	namespaceRepository := postgres.NewNamespaceRepository(dbClient)
 	namespaceService := namespace.NewService(namespaceRepository)
 
-	s := schema.NewSchemaMigrationService(
-		blob.NewSchemaConfigRepository(resourceBlobFS),
-		namespaceService,
-		roleService,
-		actionService,
-		policyService,
-		policySpiceRepository,
-	)
-
-	err = s.RunMigrations(ctx)
-	if err != nil {
-		return err
-	}
-
 	deps, err := buildAPIDependencies(logger, cfg, resourceBlobRepository, dbClient, spiceDBClient)
 	if err != nil {
 		return err
 	}
+	// load metadata schema in memory from db
+	if err := deps.MetaSchemaService.InitMetaSchemas(context.Background()); err != nil {
+		logger.Warn("metaschemas initialization failed", "err", err)
+	}
```

Comment:
I think we should move this to `shield server migrate` command. We won't be needing this once done.

Reply:
Have split this function into two. InitMetaSchemas will be required whenever the server starts. This function populates the in-memory cache with the existing metaschemas in the database.

MigrateDefault function being used to populate the default schemas will run only during migration.